src/filter.ts

```diff
@@ -156,7 +160,7 @@ export class Filter {
    * //   startTest2Exclusive: 'value3'
    * // }
    */
-  static createRange(start, end, key) {
+  static createRange(start, end, key: string) {
```

Comment:
looks like... Pascal. What are the types for `start` and `end`? :)

Reply:
Added some cheap well know ones first.  This library is a trip man.  Will come back around for completeness again.